checker/src/org/checkerframework/checker/i18nformatter/I18nFormatterTransfer.java

```diff
@@ -63,7 +63,8 @@ public I18nFormatterTransfer(I18nFormatterAnalysis analysis, I18nFormatterChecke
                     thenStore.insertValue(firstParam, anno);
                 }
             } catch (FlowExpressionParseException e) {
-                // errors are reported at declaration site
+                // report errors here
+                checker.report(e.getResult(), node);
```

Comment:
Ok I see now.  The user shouldn't see this warning ever.  I would ErrorAbort instead. (same for FormatTransfer)